log_test.go

```diff
@@ -1039,3 +1039,22 @@ func TestUnmarshalTextLevel(t *testing.T) {
 		})
 	}
 }
+
+func TestFatalFunc(t *testing.T) {
```

Comment:
Could you also create a test case to make sure the fatal function can be passed down to the child logger created by a `With()` function?

Reply:
Added.